src/js/components/App.js

```diff
@@ -1,14 +1,16 @@
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
```

Comment:
add empty line after

Reply:
Do you want to separate custom components from libraries?